src/runtime_src/core/tools/xbmgmt2/SubCmdConfigure.cpp

```diff
@@ -183,46 +183,54 @@ show_device_conf(xrt_core::device* device)
   if (is_mfg || is_recovery)
     throw xrt_core::error(std::errc::operation_canceled, "This operation is not supported with manufacturing image.\n");
 
+  boost::format sysfs_format("  %-33s: %s\n");
```

Comment:
sysfs is a linux specific term, can you rename it to something more generic like fmt_output